pkg/product/mke/phase/uninstall_mcr.go

```diff
@@ -27,14 +28,46 @@ func (p *UninstallMCR) Run() error {
 	return nil
 }
 
-func (p *UninstallMCR) uninstallMCR(h *api.Host, c *api.ClusterConfig) error {
+func (p *UninstallMCR) uninstallMCR(h *api.Host, config *api.ClusterConfig) error {
```

Comment:
perhaps this function belongs in https://github.com/Mirantis/mcc/blob/main/pkg/mcr/mcr.go as an abstract mcr feature.

Reply:
you could also move the vomume prune to the mcr.go, but I won't block this merge.